### PR TITLE
[stable8] Backport scrollbar fixes

### PR DESCRIPTION
**Pull request by MorrisJobke**
Backport of #15260 

Approved here: https://github.com/owncloud/core/pull/15260#issuecomment-108871947

@owncloud/designers @PVince81 please review

### CODE REVIEW

File: core/js/js.js [hunk 1]
```diff
@@ -1615,3 +1642,31 @@ jQuery.fn.selectRange = function(start, end) {
 jQuery.fn.exists = function(){
 	return this.length > 0;
 };
+
+function getScrollBarWidth() {
```
Comment (PVince81) [inline core/js/js.js]: Arghh, global namespace pollution.

Next time please use `OC._getScrollbarWidth` or other namespaces.
(I know this is a backport)

Comment (jancborchardt) [inline core/js/js.js]: You were cc'd to review https://github.com/owncloud/core/pull/15260 ;)

**Comment by MorrisJobke** [comment]
@PVince81 @jancborchardt Review please ;)

**Comment by jancborchardt** [comment]
Well I can’t really review my own commits, eh? ;D @owncloud/designers @davitol @jnfrmarks

**Comment by MorrisJobke** [comment]
@jancborchardt But you can test this again, because I checked it too (at the time of the backport) and together we have a single plus one ;)

**Comment by DeepDiver1975** [comment]
@jancborchardt @PVince81 @MorrisJobke let's roll this in - please review/test/fix/what-ever - THX

**Comment by MorrisJobke** [comment]
> @MorrisJobke let's roll this in - please review/test/fix/what-ever - THX

I already tested this and it works. Needs another reviewer. @PVince81 @LukasReschke want to review this?

**Comment by PVince81** [comment]
:shipit: :+1:

**Comment by davitol** [comment]
:+1:

**Pull request merged**